### ISSUE TITLE
[Bug]: Mushroom Chip Card using light/dark mode color

### ISSUE THREAD

**Issue by HA-Roberto** [piitaya/lovelace-mushroom#609]
### Current Behavior

The HA theme mode (light or Dark) is overwriting the mushroom chip icon color. Setting the color on the chip via UI has no affect and chips that support automatic color like lights do not change. 

Inspecting the icon shows that there are two color styles being applied. The actual correct color is one of them, but it seems to be trumped by the mode color 'rgb(225,225,225)' or 'rgb(33,33,33)'
![image](https://user-images.githubusercontent.com/80992882/178736944-bbfb37d2-eaf5-4ae9-a305-f9b0b3587e23.png)

![image](https://user-images.githubusercontent.com/80992882/178737644-04cd2da8-4cb6-49c2-81bb-0a90b46e2a25.png)
![image](https://user-images.githubusercontent.com/80992882/178737718-cc497d2b-f9e0-4328-9093-17de0948e60f.png)

### Expected Behavior

Icon for the mushroom chip should be colored depending on the entity type and state. 

### Steps To Reproduce

This is happening on all dashboards, even if I create a new one. I've tried two different browsers, cleared cache and cookies, changed themes, removed the themes configuration from my configuration.yaml, manually tried to set icon color via UI or template chip, reinstalled mushroom icons via HACS, and restarted HA instance.

### Environment

```markdown
- OS: HA OS
- Browsers tested on: Chrome, FireFox, HA Companion app
- Hassio:2022.7.3
```

**Comment by KTibow** [comment]
Could not reproduce. Please send your YAML config for your chips.
![image](https://user-images.githubusercontent.com/10727862/179777284-ca813ce0-839a-4894-83d2-c6d175a708f5.png)

**Comment by HA-Roberto** [comment]
I created a new dashboard to test. Here is the entire dashboard code with a few chip icons for testing.

`views:
  - title: Home
    cards:
      - type: custom:mushroom-chips-card
        chips:
          - type: template
            content: |2-
                    {% if states.light | selectattr('state', 'eq', 'on')
                      | map(attribute='entity_id') | map('area_name')
                      | select('string')
                      | list | count  == 0 %}
                     {0}
                    {%else %}
              {{states.light | selectattr('state', 'eq', 'on')
                      | map(attribute='entity_id') | map('area_name')
                      | select('string')
                      | list | count}}
                    {%- endif %}
            icon: mdi:lightbulb
            icon_color: |
              {% if       
                          (states.light | selectattr('state', 'eq', 'on')
                          | map(attribute='entity_id') | map('area_name')
                          | select('string')
                          | list | count) >= 1 -%}
              red
              {%- else -%}
              blue
              {%- endif %}
            hold_action:
              action: call-service
              service: light.turn_off
              service_data: {}
              target:
                area_id:
                  - 2_bedroom
                  - back_entrance
                  - dining_room
                  - front_entrance
                  - garage
                  - 1_bedroom
                  - kitchen
                  - living_room
                  - bedroom
                  - office
          - type: entity
            entity: sensor.average_indoor_temperature
            use_entity_picture: false
            icon: mdi:home-thermometer
            icon_color: purple
          - type: alarm-control-panel
            entity: alarm_control_panel.home_alarm
          - type: entity
            entity: lock.front_door_lock
            name: Door
            tap_action:
              action: more-info
          - type: light
            entity: light.office_ambient
            use_light_color: true
          - type: entity
            entity: sensor.panel_temperature
            icon_color: red
        alignment: center
`

![image](https://user-images.githubusercontent.com/80992882/179843187-0d89f24c-6ec6-438b-8c5f-1e36e05c408c.png)

**Comment by KTibow** [comment]
Odd. Have you tried a ctrl+shift+r on default theme yet?
Works for me with
```yaml
type: custom:mushroom-chips-card
chips:
  - type: entity
    entity: sensor.family_room_temperature
    icon_color: red
```

**Comment by HA-Roberto** [comment]
Same thing. I just swapped over to a different PC and tried as well with no luck.
Changed to the default theme using Auto mode (Firefox Browser using a dark theme) and created that same card you added but used my own sensor entity and had the same results. 

The only thing I've seen that is able to change the colors is changing the mode from light mode to dark mode. 

Dark mode: results in
`<ha-icon style="--color: rgb(var(--rgb-red)); color: rgb(225, 225, 225);" class=" active "></ha-icon>`

Light mode: results in
`<ha-icon style="--color: rgb(var(--rgb-red)); color: rgb(33, 33, 33);" class=" active "></ha-icon>`

And all of the chip cards have that "color: rgb(33, 33, 33);" or "color: rgb(225, 225, 225)" regardless if im manually setting the color via UI, a jinja template, or if its defaulting based on the cards logic.

**Comment by piitaya** [comment]
Maybe you have a theme or a card that override the color ?

**Comment by HA-Roberto** [comment]
How could I check this? I've tried changing the theme from the drop down list and all of them have the same effect. I've also tried removed the theme folder from the configuration.yaml and restarted my instance with the no luck. 
The real weird thing is that I've only been able to see this issue on the mushroom-chip card.

**Comment by piitaya** [comment]
Which custom cards are installed with HACS? Can you list them?

**Comment by HA-Roberto** [comment]
I've figured it out and and I'm sorry for the time wasted! 
It seems like there was an issue with my card-mod installation. I started the process of removing cards and re-installing them. I dont know what exactly changed but I removed my card-mod card (which broke a lot of things...) and then when I re-installed it, I had color!!! 

I currently have the latest version of card-mod that is available through HACS 3.1.5. There is a chance I had an older version even though HACS hadn't told me I had an update, but I didnt check the version number so I am not sure if that was the case.

Thank you for working with me through this.
